src/redux/configStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunkMiddleware from 'redux-thunk';
+import countriesReducer from './home';
+
+const reducer = combineReducers({
+  countriesReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger, thunkMiddleware),
+);
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.

Reply:
Fixed

Reply:
Dear @oloomoses, 🥇 

I will fix your notes a.s.a.p.

Thank you for the review! 👍 👍 👍 

My best regards,
Elmar